GitUI/BranchTreePanel/RepoObjectsTree.Nodes.cs

```diff
@@ -218,8 +219,37 @@ public virtual string DisplayText()
                 return ToString();
             }
 
+            protected void SetNodeFont(FontStyle style)
+            {
+                if (style == FontStyle.Regular)
```

Comment:
Maybe good to avoid using null (or Default for a new type), but it also will make it impossible to force regular. Should be OK.

Reply:
I'm not sure I understand what you mean here. Are you saying this is okay?

Reply:
Yes, thinking loud.